src/allroadstoliterature/main.py

```diff
@@ -1,42 +1,36 @@
-import pprint
```

Comment:
question to self: `tests/test_main.py`  imports `from allroadstoliterature.main import get_doi_metadata`... will it still work?